File: software/examples/python/example_simple_voltage.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

HOST = "localhost"
PORT = 4223
UID = "XYZ" # Change XYZ to the UID of your Industrial Analog Out Bricklet 2.0

from tinkerforge.ip_connection import IPConnection
from tinkerforge.bricklet_industrial_analog_out_v2 import BrickletIndustrialAnalogOutV2

if __name__ == "__main__":
    ipcon = IPConnection() # Create IP connection
    iao = BrickletIndustrialAnalogOutV2(UID, ipcon) # Create device object

    ipcon.connect(HOST, PORT) # Connect to brickd
    # Don't use device before ipcon is connected

    # Set output voltage to 3.3V
    iao.set_voltage(3300)
    iao.enable()

    raw_input("Press key to exit\n") # Use input() in Python 3
    iao.disable()
    ipcon.disconnect()

File: software/examples/go/example_simple_voltage.go
package main

import (
	"fmt"
	"tinkerforge/industrial_analog_out_v2_bricklet"
	"tinkerforge/ipconnection"
)

const ADDR string = "localhost:4223"
const UID string = "XYZ" // Change XYZ to the UID of your Industrial Analog Out Bricklet 2.0.

func main() {
	ipcon := ipconnection.New()
	defer ipcon.Close()
	iao, _ := industrial_analog_out_v2_bricklet.New(UID, &ipcon) // Create device object.

	ipcon.Connect(ADDR) // Connect to brickd.
	defer ipcon.Disconnect()
	// Don't use device before ipcon is connected.

	// Set output voltage to 3.3V
	iao.SetVoltage(3300)
	iao.SetEnabled(true)

	fmt.Print("Press enter to exit.")
	fmt.Scanln()

	iao.SetEnabled(false)
}
